internal/worker/worker.go

```diff
@@ -100,7 +102,7 @@ func (w *BaseWorker) Start(ctx context.Context) error {
 		case cmdReq := <-w.adhocReqChan:
 			// Handle adhoc requests of DiceDBCmd
 			func() {
-				execCtx, cancel := context.WithTimeout(ctx, 6*time.Second) // Timeout set to 6 seconds for integration tests
+				execCtx, cancel := context.WithTimeout(ctx, 500*time.Second) // Timeout set to 6 seconds for integration tests
```

Comment:
500 seconds seems to be very high. Should we keep it to the minimum?

**Note**: We should ideally have the timeout configurable and have it higher for test-cases.